File: 코드샘플/메타토이드래곤즈/유니티/System/Data/Datas/WorldData.cs
using Newtonsoft.Json.Linq;
using System.Collections.Generic;

namespace SandboxNetwork
{
    public class WorldData : TableData<DBWorld_base>
    {
        static private WorldTable table = null;
        static public WorldData Get(string key)
        {
            if (table == null)
                table = TableManager.GetTable<WorldTable>();

            return table.Get(key);
        }
        static public WorldData GetByWorldNumber(int worldNumber)
        {
            if (table == null)
                table = TableManager.GetTable<WorldTable>();

            return table.GetByWorldNumber(worldNumber);
        }

        public int NUM => Data.NUM;
        public int _NAME => Data._NAME;
        public string BACKGROUND => Data.BACKGROUND;
        public string IMAGE => Data.IMAGE;

        List<int> star = null;
        List<int> reward = null;
        public List<int> STAR
        {
            get { 
                if(star == null)
                {
                    star = new List<int>();
                    star.Add(Data.STAR_1);
                    star.Add(Data.STAR_2);
                    star.Add(Data.STAR_3);
                }

                return star;
            }
        }
        public List<int> STAR_REWARD
        {
            get
            {
                if (reward == null)
                {
                    reward = new List<int>();
                    reward.Add(Data.REWARD_STAR_1);
                    reward.Add(Data.REWARD_STAR_2);
                    reward.Add(Data.REWARD_STAR_3);
                }

                return reward;
            }
        }

    }
}

File: 코드샘플/메타토이드래곤즈/코코스크리에이터/Scripts/Data/WorldData.ts

import { _decorator } from 'cc';
import { DataBase } from './DataBase';

export class WorldData extends DataBase 
{
    protected num: number = -1;
    public get NUM(): number {
        return this.num;
    }
    public set NUM(value: number) {
        this.num = value;
    }

    protected name: number = -1;
    public get NAME(): number {
        return this.name;
    }
    public set NAME(value: number) {
        this.name = value;
    }

    protected background: string = "";
    public get BACKGROUND(): string {
        return this.background;
    }
    public set BACKGROUND(value: string) {
        this.background = value;
    }

    protected image: string = "";
    public get IMAGE(): string {
        return this.image;
    }
    public set IMAGE(value: string) {
        this.image = value;
    }

    protected star: number[] = Array<number>(3);
    public get STAR(): number[] {
        return this.star;
    }
    public set STAR(value: number[]) {
        this.star = value;
    }

    protected star_rewrad: number[] = Array<number>(3);
    public get STAR_REWARD(): number[] {
        return this.star_rewrad;
    }
    public set STAR_REWARD(value: number[]) {
        this.star_rewrad = value;
    }
}